src/app/static/src/app/components/downloadResults/DownloadResults.vue

```diff
@@ -112,27 +120,31 @@
         downloadSummaryReport: () => void
         downloadSpectrumOutput: () => void
         downloadCoarseOutput: () => void
+        downloadComparisonReport: () => void
         downloadUrl: (downloadId: string) => string
         handleDownloadResult: (downloadResults: DownloadResultsDependency) => void
     }
 
     interface Data {
-        uploadModalOpen: boolean
+        uploadModalOpen: boolean,
+        comparisonSwitch: boolean
     }
 
     export default Vue.extend<Data, Methods, Computed>({
         name: "downloadResults",
         data() {
             return {
-                uploadModalOpen: false
+                uploadModalOpen: false,
+                comparisonSwitch: !switches.comparisonOutput
```

Comment:
Why is this doing a negation? Show comparison view should be true if the switch is true, shouldn't it?